specs/mgt-file-picker.md

```diff
@@ -9,49 +9,41 @@ The file picker component provides a button, a drop down list of files, and as w
 | Feature | Priority | Notes |
 | ------- | -------- | ----- |
 | Provide a button that renders a drop down when clicked | P0 | |	
-| Retrieve a list of files based on information provided | P0 | |
-| Use the `mgt-file-list` component to render the list inside of the drop down | P0 | |
-| On selection or deselection of a file, an event is fired | P0 | |	
+| Retrieve files based on specified insight type | P0 | |
+| Use `mgt-file` components to render each file inside of the drop down | P0 | |
+| On selection of a file, a `selectionChanged` event is fired | P0 | |	
 | The selected file object(s) is accessible by the developer for further action |P0 | |	
-| Ability for developer to toggle between single-select and multiselect | P1 |  | 
+| Extended file picker is launched when “See all” is clicked | P0 | |
+| Ability for developer to toggle between single-select and multiselect | P1 | Will not be included in v1. | 
 | Visual indication of selected files | P2 | Will not be included in v1. |
 | Render the selected files | P2 | Will not be included in v1. We will provide a sample for developers to render selected files using `mgt-file` components themselves. |
-| Extended file picker is launched when “See all” is clicked | P2 | Not included in v1. |
 
 ## Proposed Solution
 
 ### Example 1: Show the user's top 10 most recently used files to select from
 ```<mgt-file-picker insight-type=“used” show-max=“10”></mgt-file-picker>```
 
-### Example 2: Show a specific folder in OneDrive to select files from
-```<mgt-file-picker drive-id=“123” item-id=“456”></mgt-file-picker>```
-
 ## Attributes and Properties
 
 | Attribute | Property | Description |
 | --------- | -------- | ----------- |
-| `file-list-query` | `fileListQuery` | The full query or path to the folder in a drive or site to retrieve the list of files from |
```

Comment:
I wonder if we could still keep these even though they are not planned for v1? Could we mark them somehow as beyond v1?

Reply:
Added them back in and separated between v1 and v2 in the tables